L/LinearFold/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase, Pkg
+
+name = "LinearFold"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/LinearFold/LinearFold/archive/refs/tags/v$(version.major).$(version.minor).tar.gz",
+                  "2ae56b5f183472c2de96782e770a91e57f82e0ab511dfc0d9d612aa4e6155f60"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/LinearFold-*/
+
+make -j${nproc}
+mkdir -p ${bindir}
+for prg in linearfold_c linearfold_v; do
+    cp bin/$prg ${bindir}/${prg}${exeext}
```

Comment:
I prefer to use `install`:
```suggestion
for prg in linearfold_c linearfold_v; do
    install -Dvm 755 "bin/${prg}" "${bindir}/${prg}${exeext}"
```